File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/06_compito_java_20150917/InCasa.py
from Partita import Partita


class InCasa(Partita):
    def __init__(self, tipo, nome_palazzetto, data, ora, risultato, vinta):
        super().__init__(tipo, data, ora, risultato, vinta)
        self._nome_palazzetto = nome_palazzetto

    def __str__(self):
        return self._nome_palazzetto + "\t-\t" + super().__str__()

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/06_compito_java_2015_09_17/InCasa.java
import java.util.*;

public class InCasa extends Partita {
    private String palazzetto;

    public InCasa(String data, String ora, String risultato, int vittoria, String palazzetto) {
        super(data, ora, risultato, vittoria);
        this.palazzetto = palazzetto;
    }

    public String toString() {
        return palazzetto + "\t-\t" + data + "\t" + ora + "\t" + risultato;
    }
}